staging/src/k8s.io/api/go.mod

```diff
@@ -6,11 +6,14 @@ go 1.12
 
 require (
 	github.com/gogo/protobuf v1.2.2-0.20190723190241-65acae22fc9d
-	github.com/stretchr/testify v1.3.0
+	github.com/stretchr/testify v1.4.0
 	k8s.io/apimachinery v0.0.0
 )
 
 replace (
+	github.com/go-openapi/spec => github.com/go-openapi/spec v0.19.2
+	github.com/mailru/easyjson => github.com/mailru/easyjson v0.0.0-20190626092158-b2ccc519800e
+	github.com/stretchr/testify => github.com/stretchr/testify v1.3.0
```

Comment:
need to satisfy `hack/lint-dependencies.sh` so our pinned versions match naturally preferred versions

Reply:
done